Library/Formula/bzr-colo.rb

```diff
@@ -0,0 +1,53 @@
+class BzrColo < Formula
+  desc "Git style colocated branches for Bazaar"
+  homepage "https://launchpad.net/bzr-colo"
+  url "https://launchpad.net/bzr-colo/trunk/0.4.0/+download/bzr-colo-0.4.0.tar.gz"
+  sha256 "f61c1abaf80f1e4a573fefd492b70938d27c4b8ca5611cdb0e0a4dc0ed71bbb3"
+
+  bottle :unneeded
+  depends_on "bazaar"
+
+  def install
+    (share/"bazaar/plugins/colo").install Dir["*"]
+  end
+
+  test do
+    modify_file = "echo \"change\" >> foo.txt"
+
+    # Set up repository with colocated branches
+    system "bzr", "whoami", "Homebrew"
+    system "bzr", "colo-init", "foo"
+
+    cd "foo" do
+      # Checkout a working tree
+      system "bzr", "colo-checkout", "trunk"
+
+      cd "trunk" do
+        # Commit a change in the trunk branch
+        system modify_file
```

Comment:
def write(content, *open_args)
    raise "Will not overwrite #{self}" if exist?
    dirname.mkpath
    open("w", *open_args) { |f| f.write(content) }
  end


It looks like I can’t write to the same file twice.

- Fred McCann
fred@sharpnoodles.com

> On Feb 14, 2016, at 11:59 AM, Mike McQuaid <notifications@github.com> wrote:
> 
> In Library/Formula/bzr-colo.rb <https://github.com/Homebrew/homebrew/pull/49097#discussion_r52845455>:
> 
> > +  end
> > +
> > +  test do
> > +    modify_file = "echo \"change\" >> foo.txt"
> > +
> > +    # Set up repository with colocated branches
> > +    system "bzr", "whoami", "Homebrew"
> > +    system "bzr", "colo-init", "foo"
> > +
> > +    cd "foo" do
> > +      # Checkout a working tree
> > +      system "bzr", "colo-checkout", "trunk"
> > +
> > +      cd "trunk" do
> > +        # Commit a change in the trunk branch
> > +        system modify_file
> Use (testpath/"foo.txt").write "change") instead.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Homebrew/homebrew/pull/49097/files#r52845455>.
>